docker-compose.yml

```diff
@@ -488,17 +488,36 @@ services:
     shm_size: *shm-size
     ulimits: *ulimits
     environment: &cuda-environment
-      <<: *ccache
+      <<: [*sccache, *ccache]
       ARROW_BUILD_STATIC: "OFF"
       ARROW_CUDA: "ON"
       ARROW_GANDIVA: "OFF"
       ARROW_GCS: "OFF"
       ARROW_ORC: "OFF"
       ARROW_S3: "OFF"
       ARROW_SUBSTRAIT: "OFF"
+      ARROW_BUILD_UTILITIES: "OFF"
+      ARROW_COMPUTE: "OFF"
+      ARROW_CSV: "OFF"
+      ARROW_DATASET: "OFF"
+      ARROW_FILESYSTEM: "OFF"
+      ARROW_HDFS: "OFF"
+      ARROW_JEMALLOC: "OFF"
+      ARROW_JSON: "OFF"
+      ARROW_PLASMA: "OFF"
+      ARROW_PARQUET: "OFF"
+      ARROW_ENABLE_TIMING_TESTS: "OFF"
       ARROW_WITH_OPENTELEMETRY: "OFF"
     volumes: *ubuntu-volumes
-    command: *cpp-command
+    command: /bin/bash -c "
+        /arrow/ci/scripts/cpp_build.sh /arrow /build &&
+        /arrow/ci/scripts/cpp_test.sh /arrow /build"
```

Comment:
It seems that this is (logically) same as `cpp-command`. Why can't we use `*cpp-command` here?